src/rabbit_fifo.erl

```diff
@@ -268,9 +273,11 @@ update_config(Conf, State) ->
     DLH = maps:get(dead_letter_handler, Conf, undefined),
     BLH = maps:get(become_leader_handler, Conf, undefined),
     SHI = maps:get(shadow_copy_interval, Conf, ?SHADOW_COPY_INTERVAL),
+    SingleActiveConsumerOn = maps:get(single_active_consumer_on, Conf, false),
```

Comment:
`single_active_consumer_on` would need to be added to the `config()` type above.

Reply:
Done.